src/main/groovy/au/org/ala/ecodata/paratoo/ParatooProject.groovy

```diff
@@ -25,6 +29,31 @@ class ParatooProject {
     Map projectArea = null
     Site projectAreaSite = null
     List<Site> plots = null
+    Program program
+
+    private Map getConfig() {
+        Map config = program?.getInheritedConfig() ?: [:]
+        // Support per-project overrides of config
+        if (project.config) {
+            config.putAll(project.config)
+        }
+        config
+    }
+
+    List<String> getDefaultModules() {
+        Map config = getConfig()
+        List modules = DEFAULT_MODULES
+        if (config.hasProperty(PARATOO_DEFAULT_MODULES)) {
```

Comment:
I don't think `hasProperty` will work here. Groovy doco for hasProperty says `Returns true of the implementing MetaClass has a property of the given name`. I faced a similar issue in the past.

Reply:
Thanks for picking that up Temi, I've replaced hasProperty with containsKey